app/services/step_factory.rb

```diff
@@ -90,10 +14,11 @@ def message
   class << self
     def create(name)
       classified_name = name.camelize
-      # need to redirect to root or last valid step.
-      raise NameNotFoundError, classified_name unless STEP_NAMES.include?(classified_name)
 
       Object.const_get(classified_name).new
```

Comment:
This will now instantiate _any_ ruby class in the system its told to - feels a bit open ended?